setup.py

```diff
@@ -1,21 +1,23 @@
 """Setup script for lasio"""
 
-import lasio
 import re
 from setuptools import setup
 import os
 
-__init__file = os.path.join('lasio', '__init__.py')
-with open(__init__file, 'r') as f:
-    file_contents = f.read()
+if 'LASIO_VERSION' in os.environ:
```

Comment:
add else and raise an exception there. In case somebody breaks your changes in `leapfrog-build-scripts` it will remind them

Reply:
I am adding an `ensure` call on `leapfrog-build_scripts` to make sure it "succeeded" or the version is set correctly.